worker/uniter/runner/jujuc/state-set_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package jujuc_test
+
+import (
+	"bytes"
+
+	"github.com/juju/cmd"
+	"github.com/juju/cmd/cmdtesting"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/worker/uniter/runner/jujuc"
+)
+
+type stateSetSuite struct {
+	stateSuite
+}
+
+var _ = gc.Suite(&stateSetSuite{})
+
+func (s *stateSetSuite) TestHelp(c *gc.C) {
+	toolCmd, err := jujuc.NewCommand(nil, "state-set")
+	c.Assert(err, jc.ErrorIsNil)
+
+	ctx := cmdtesting.Context(c)
+	code := cmd.Main(jujuc.NewJujucCommandWrappedForTest(toolCmd), ctx, []string{"--help"})
+	c.Check(code, gc.Equals, 0)
+	c.Assert(bufferString(ctx.Stderr), gc.Equals, "")
+
+	var expectedHelp = `
+Usage: state-set [options] key=value [key=value ...]
+
+Summary:
+set server-side-state values
+
+Options:
+--file  (= )
+    file containing key-value pairs
+
+Details:
+state-set sets the value of the server side state specified by key.
+
+The --file option should be used when one or more key-value pairs are
+too long to fit within the command length limit of the shell or
+operating system. The file will contain a YAML map containing the
+settings.  Settings in the file will be overridden by any duplicate
```

Comment:
Should we explicitly mention that the _values_ in the yaml document can only be strings? I suspect someone will try to call state-set with a file like:

```yaml
foo: bar
boo:
  -1
  -2
```

Reply:
done